fdbserver/include/fdbserver/ClusterController.actor.h

```diff
@@ -2960,21 +2965,35 @@ class ClusterControllerData {
 				workerHealth[req.address].degradedPeers[degradedPeer] = { currentTime, currentTime };
 			}
 
+			// TODO(zhewu): add disconnected peers in worker health.
+			for (const auto& degradedPeer : req.disconnectedPeers) {
+				workerHealth[req.address].degradedPeers[degradedPeer] = { currentTime, currentTime };
+			}
+
 			return;
 		}
 
 		// The incoming worker already exists in `workerHealth`.
 
 		auto& health = workerHealth[req.address];
 
-		// Update the worker's degradedPeers.
-		for (const auto& peer : req.degradedPeers) {
-			auto it = health.degradedPeers.find(peer);
+		auto updateDegradedPeer = [&health, &currentTime](const NetworkAddress& degradedPeer) {
```

Comment:
Since we are not changing `currentTime` im lambda, I would prefer either make line 2958 `const double currentTime = now()`, or pass `currentTime` by value in capture.

Reply:
Done